tests/SideBySide.New/ConnectionPool.cs

```diff
@@ -80,37 +80,7 @@ public void ResetConnection(object connectionReset, int poolSize, long expected)
 		}
 
 		[Fact]
-		public async Task ExhaustConnectionPoolWithTimeout()
```

Comment:
Ok, we should reimplement this using a timeout CancellationToken inside of `ConnectionPool.GetSessionAsync` then